lib/src/widgets/chat/ds_message_bubble.widget.dart

```diff
@@ -1,26 +1,29 @@
 import 'package:blip_ds/src/enums/ds_border_radius.enum.dart';
 import 'package:blip_ds/src/enums/ds_align.enum.dart';
 import 'package:blip_ds/src/themes/colors/ds_colors.theme.dart';
+import 'package:blip_ds/src/widgets/animations/ds_animated_size.widget.dart';
 import 'package:flutter/material.dart';
 
 class DSMessageBubble extends StatelessWidget {
   final DSAlign align;
   final Widget child;
-  final EdgeInsets contentPadding;
   final List<DSBorderRadius> borderRadius;
+  final EdgeInsets padding;
+  final bool groupWithPreviousMessage;
 
   const DSMessageBubble({
     Key? key,
     required this.align,
     required this.child,
     this.borderRadius = const [DSBorderRadius.all],
-    this.contentPadding = const EdgeInsets.symmetric(
-      vertical: 10.0,
-      horizontal: 14.0,
+    this.padding = const EdgeInsets.symmetric(
+      vertical: 10,
```

Comment:
It'll conflict with Andre's commit. The correct is `vertical: 8` and `horizontal: 16`. You can accept the incoming change

Reply:
Sure, and there are even more files that will conflict. That's why I decided to be the last to merge. Anyway, thanks for the advice.